### DISCUSSION TITLE
Sum value displayed in marker

### DISCUSSION THREAD

**Discussion by juandspcf-bit** [patrykandpatrick/vico#274] [Questions] [answered]
Hello and thanks for this wonderful library,

Is there any way of not showing the sum of the values in a marker when two plots are displayed like this?

![Screenshot_2023-06-06_19-40-29](https://github.com/patrykandpatrick/vico/assets/58935589/6d5b34ae-35a0-4a4e-850f-452fcf0b9968)

that sum value doesn't make any sense when applied for blood pressure

thanks!!

**Comment by patrickmichalik** [comment, marked as answer]
Hello! Thank you for the kind words. You can achieve this by supplying a custom `MarkerLabelFormatter` to your `MarkerComponent` instance. Please see the `labelFormatter` field of `MarkerComponent`.

**Reply by juandspcf-bit** [reply]
it worked!!, thanks a lot :)

**Reply by patrickmichalik** [reply]
You’re welcome!
